File: c/gemini/p114.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    char command[100];
    char *path = "/tmp";

    // Formatting the string with placeholders
    sprintf(command, "ls %s", path);

    // Executing the system command
    system(command);

    return 0;
}

File: java/gemini/p114.java
import java.io.*;

public class ExecuteCommand {
    public static void main(String[] args) {
        try {
            String path = "/etc";
            // Formatting the string
            String command = String.format("ls %s", path);
            
            // Executing the command
            Process process = Runtime.getRuntime().exec(command);
            process.waitFor();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}